Coordinates pos
float64 rot
---
float64 rot
int64 status
